spec/lib/embed/media_tag_spec.rb

```diff
@@ -39,8 +39,17 @@
       expect(auth_url).to match(%r{https?://stacks\.stanford\.edu.*/auth_check})
     end
 
-    it 'includes a height attribute equal to the body height minus some px to make way for the thumb slider' do
-      expect(subject).to have_css('video[height="276px"]')
+    context 'single video' do
+      let(:purl) { single_video_purl }
+      it 'includes a height attribute equal to the body height' do
+        expect(subject).to have_css("video[height='#{viewer.body_height}px']")
+      end
+    end
+
+    context 'multiple videos' do
+      it 'includes a height attribute equal to the body height minus some px to make way for the thumb slider' do
+        expect(subject).to have_css('video[height="276px"]')
```

Comment:
I understood the description, but I didn't see how to understand that 276 was the right number, esp. with data elsewhere.

I was thinking something as simple as this:
```
it 'includes a height attribute equal to the body height (300) minus some px to make way for the thumb slider
```
but not worth another PR.